packages/ignition/src/mdx-components.tsx

```diff
@@ -1,13 +1,31 @@
+import { HtmlHTMLAttributes } from "react";
 import type { MDXComponents } from "mdx/types";
 import Title from "@/components/documentation/title";
 import Title2 from "@/components/documentation/title2";
 import Title3 from "@/components/documentation/title3";
 import Title4 from "@/components/documentation/title4";
 import Paragraph from "@/components/documentation/paragraph";
-import Code from "@/components/documentation/code";
-import CodeStyler from "@/components/code-block/code-styler";
+import { Lang, WithCopy, CodeStyler } from "@rocketclimb/code-block";
+import Code from "@rocketclimb/code-block/code";
```

Comment:
Can't we export `Code` from `@rocketclimb/code-block` too so we can use a single line import?